lib/sycamore/sycamore/transforms/detr_partitioner.py

```diff
@@ -252,6 +466,7 @@ def infer(
                 result["scores"] = result["scores"].tolist()
                 result["labels"] = result["labels"].tolist()
                 result["boxes"] = result["boxes"].tolist()
+
```

Comment:
Intended?

Reply:
Yes. Not having a blank line before the outdent is less readable to me.